memorious/operations/aleph.py

```diff
@@ -10,21 +13,20 @@
 from memorious.core import get_rate_limit  # type: ignore
 
 
-def aleph_emit(context, data):
-    api = get_api(context)
-    if api is None:
-        return
-    collection_id = get_collection_id(context, api)
-    content_hash = data.get("content_hash")
+def _create_meta_languages(context, data) -> dict:
```

Comment:
It's creating more than just `languages` metadata. May be `create_document_metadata`  or something like that if these are document specific?